tests/test_ocr_runner_unit.py

```
@@ -0,0 +1,72 @@
+    def fake_run(*args, **kwargs):
+        return DummyProc(stdout=b'hello world')
+
+    monkeypatch.setattr(subprocess, 'run', fake_run)
+    p = tmp_path / 'f.pdf'
+    p.write_bytes(b'%PDF-1.4')
+    cnt = ocr_runner.pdftotext_count(str(p))
+    assert cnt == len(b'hello world')
+
+
+def test_run_ocrmypdf_success(monkeypatch, tmp_path):
+    def fake_run(args, stdout=None, stderr=None, timeout=None):
```

Ruff (0.14.10)

Missing return type annotation for private function `fake_run`

(ANN202)